pdu/Src/controlStateMachine_mock.c

```diff
@@ -431,7 +431,4 @@ HAL_StatusTypeDef mockStateMachineInit()
         return HAL_ERROR;
     }
     return HAL_OK;
-}
-
-
-
+}
```

Comment:
add a single additional line at the end. whenever you see this red symbol that's what it means